sources/packages/backend/apps/api/src/services/application-exception/application-exception.service.ts

```diff
@@ -162,15 +167,6 @@ export class ApplicationExceptionService extends RecordDataModelService<Applicat
         );
       }
 
-      if (
-        exceptionToUpdate.exceptionStatus !== ApplicationExceptionStatus.Pending
-      ) {
-        throw new CustomNamedError(
-          `Student application exception must be in ${ApplicationExceptionStatus.Pending} state to be assessed.`,
-          STUDENT_APPLICATION_EXCEPTION_INVALID_STATE,
-        );
-      }
-
```

Comment:
I was not expecting to remove the existing code, I was only expecting to add the application status restriction code in the query,  
```
.andWhere("application.applicationStatus != :applicationStatus", {
          applicationStatus: ApplicationStatus.Overwritten,})
```
If devs are okay with it, Iam good

Reply:
I didn't get the point of having `CustomNamedError` for `ApplicationExceptionStatus.Pending` and not having for `ApplicationStatus.Overwritten`.